### ISSUE TITLE
Problem when installing with homebrew

### ISSUE THREAD

**Issue by cprieto** [michaeleisel/homebrew-zld#8]
```
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
make: *** [build_homebrew
```

macOS Monterrey 12.0.1, XCode 13.1 with latest command line tools... Somehow, it does not detect the full XCode installation :|

**Comment by michaeleisel** [comment]
See https://github.com/nodejs/node-gyp/issues/569 and xcode-select

**Comment by jasikpark** [comment]
I was hoping this would work, but I get these results after running `sudo xcode-select --switch /Library/Developer/CommandLineTools` with command line tools installed:

```
❯ brew install michaeleisel/zld/zld                             
==> Downloading https://github.com/michaeleisel/zld/archive/refs/heads/ho
Already downloaded: /Users/calebjasik/Library/Caches/Homebrew/downloads/fe560f9185d8fdf96707287aa78ff219507f3143158e1c208848dc1f86503952--zld-homebrew-fixes-1.3.3.zip
==> Installing zld from michaeleisel/zld
==> make clean
==> make build_homebrew
Last 15 lines from /Users/calebjasik/Library/Logs/Homebrew/zld/02.make:
mkdir -p tapi-1100.0.11
curl -# -L https://opensource.apple.com/tarballs/tapi/tapi-1100.0.11.tar.gz | tar xz -C tapi-1100.0.11 --strip-components=1

curl -# -L https://github.com/intel/tbb/releases/download/v2020.1/tbb-2020.1-mac.tgz | tar xz
######                                                              10.3%#######################                                             36.1%################################################################## 100.0%
patch -p1 -d tapi-1100.0.11 < patches/tapi.patch
patching file include/tapi/Version.inc
################################################################## 100.0%
################################################################## 100.0%
################################################################## 100.0%
misc/checks.rb
/usr/local/opt/cmake/bin/cmake
xcodebuild -project ld/zld.xcodeproj -scheme zld -derivedDataPath build -configuration Release build ONLY_ACTIVE_ARCH=YES
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
make: *** [build_homebrew] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/michaeleisel/homebrew-zld/issues


```

**Comment by jasikpark** [comment]
Tried to follow https://github.com/nodejs/node-gyp/blob/master/macOS_Catalina.md#installing-node-gyp-using-the-xcode-command-line-tools-via-xcode-select---install but that seems to not have fixed things...
